docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -156,3 +156,24 @@ struct S
     public static void M([UnscopedRef] ref int x) { }
 }
 ```
+
+## `Microsoft.CodeAnalysis.EmbeddedAttribute` is validated on declaration
+
+***Introduced in Visual Studio 2022 version 17.13***
+
+The compiler now validates the shape of `Microsoft.CodeAnalysis.EmbeddedAttribute` when declared in source. Previously, the compiler
+would allow user-defined declarations of this attribute, but only when it didn't need to generate one itself. We now validate that:
+
+1. It must be internal
+2. It must be a class
```

Comment:
>It must be a class [](http://example.com/codeflow?start=3&length=18)

Do we check that it inherits from System.Attribute?

Reply:
It looks like the 6th condition implies this requirement, but it might be good to state this explicitly un this document. And probably in the error message.